cpp/src/barretenberg/dsl/acir_format/serde/acir.hpp

```diff
@@ -1046,7 +1006,7 @@ inline BinaryFieldOp::Add BinaryFieldOp::Add::bincodeDeserialize(std::vector<uin
     auto deserializer = serde::BincodeDeserializer(input);
     auto value = serde::Deserializable<BinaryFieldOp::Add>::deserialize(deserializer);
     if (deserializer.get_buffer_offset() < input.size()) {
-        throw_or_abort("Some input bytes were not read");
+        throw serde::deserialization_error("Some input bytes were not read");
```

Comment:
These changes are preventing bberg from compiling.

Reply:
That's weird, bberg was compiling for me